chromaprint.py

```diff
@@ -25,7 +25,7 @@ def _guess_lib_name():
     if sys.platform == 'darwin':
         return ('libchromaprint.1.dylib', 'libchromaprint.0.dylib')
     elif sys.platform == 'win32':
-        return ('chromaprint.dll', 'libchromaprint.dll')
+        return ('', 'libchromaprint.dll')
```

Comment:
Only libchromaprint.dll is created by chromaprint's build, and it contains all the used functions.